SPECS/nbdkit/nbdkit.spec

```diff
@@ -120,7 +120,7 @@ BuildRequires:  %{_bindir}/qemu-io
 BuildRequires:  %{_bindir}/qemu-nbd
 BuildRequires:  /sbin/sfdisk
```

Comment:
wondering if this should also use _sbindir?

Reply:
I was wondering the same thing, but it's coming from `util-linux`, which still says it's under `/sbin/sfdisk`. I'm sure we could fix this as well but I decided not to go down this rabbit hole at this point.